src/main/java/com/airlines/dao/CrewMemberDao.java

```diff
@@ -10,4 +10,6 @@
 
     CrewMember findById(int id);
```

Comment:
example of inconsistency and symmetry principle violation, if crewID as param, why method is findById? 

Using different approaches for same cases is violation of clean code principle.

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/80c1f66083580a28c27bea038eea1e050a97f30e